src/http/sr-modules-operators-keys/sr-modules-operators-keys.controller.ts

```diff
@@ -115,12 +115,12 @@ export class SRModulesOperatorsKeysController {
     type: TooEarlyResponse,
   })
   @Get('operators/keys')
-  async getModulesOperatorsKeysStream(@Res() reply: FastifyReply) {
+  async getModulesOperatorsKeysStream(@Query() filters: KeyQuery, @Res() reply: FastifyReply) {
```

Comment:
need e2e test :)

Reply:
Ok. I will close this MR and reopen when I have time for test.